pkg/asset/installconfig/platformpermscheck.go

```diff
@@ -8,10 +8,11 @@ import (
 
 	"github.com/openshift/installer/pkg/asset"
 	awsconfig "github.com/openshift/installer/pkg/asset/installconfig/aws"
+	gcpconfig "github.com/openshift/installer/pkg/asset/installconfig/gcp"
 	"github.com/openshift/installer/pkg/types/aws"
 	"github.com/openshift/installer/pkg/types/azure"
 	"github.com/openshift/installer/pkg/types/baremetal"
-	"github.com/openshift/installer/pkg/types/gcp"
+	gcp "github.com/openshift/installer/pkg/types/gcp"
```

Comment:
```suggestion
	"github.com/openshift/installer/pkg/types/gcp"
```
should also just work.. ?